UPGRADE-2.1.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade from 2.0 to 2.1
 
+## Sonata Admin
+
+* All Sonata Admin integration has been removed. The integration is now
+  available via the CmfSonataAdminIntegrationBundle.
```

Comment:
should we say "All Sonata Admin integrations have been moved to the CmfSonataAdminIntegrationBundle."? just so the shock of "has been removed" can be avoided...